paths_cli/wizard/openmm.py

```diff
@@ -61,7 +61,12 @@
             ProxyParameter(
                 name='n_frames_max',
                 ask="How many frames before aborting a trajectory?",
-                helper=None,
+                helper=("Sometimes trajectories can get stuck in "
+                        "unexpected basins. To prevent your trajectory "
+                        "from running forever, you need to add a cutoff "
+                        "trajectory length. This should be significantly "
+                        "longer than you would expect a transition to "
+                        "take."),
```

Comment:
you might want to add a sensible default (for example 100x the expected transition length)

Reply:
I thought about this, but I don't think there's a totally obvious default value. It really depends on the underlying PES. It's really just "at what point should a metastable well be treated as a stable state," and the line between a metastable well in the barrier region and an additional state in MSTPS is not really well-defined.